src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -895,11 +895,11 @@ class JsonLinePrinter : public JsonPrinter {
         BSLS_KEYWORD_OVERRIDE
     {
         closeBraceIfOpen();
-        d_ostream << "  \"TotalRecordsNumber\": \"" << totalRecordsCount
-                  << "\",\n";
+        d_ostream << "  \"totalRecordsNumber\": " << totalRecordsCount
+                  << ",\n";
 
         // Print information per Queue:
-        d_ostream << "  \"PerQueueRecordsNumber\": [\n";
+        d_ostream << "  \"perQueueRecordsNumber\": [\n";
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.